projects/vdk-core/src/vdk/internal/cli_entry.py

```diff
@@ -125,15 +125,18 @@ def vdk_main(
         configuration = build_configuration(plugin_registry)
         core_context = build_core_context_and_initialize(configuration, plugin_registry)
 
-        exit_code = 0
+        exit_code = 1
```

Comment:
nit: maybe have the comment you had previously provided, explaining why 1 in case no root_command